source/Information.h

```diff
@@ -54,13 +56,15 @@ class Information {
 	const Color &GetOutlineColor() const;
 
 
+
```

Comment:
```suggestion
```

Reply:
I must not fully understand the coding standards.

Reply:
> I must not fully understand the coding standards.

Should be two or one lines between sections (public, protected, private), with the trend being two.